src/BlazorUI/Bit.BlazorUI/Components/Utilities/Text/BitText.scss

```
@@ -138,11 +144,135 @@
 }
 
+// The transparent foreground is the gradient text technique: the box paints a background and the glyphs are
+// punched out of it. A forced-colors palette removes every gradient it would have shown through, so the fill
+// is given back to the system color there - otherwise the text is painted with nothing behind it.
 .bit-txt-rfg {
     color: transparent;
+
+    @media (forced-colors: active) {
+        color: CanvasText;
+        background-image: none;
+        -webkit-text-fill-color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)